app/Http/routes.php

```diff
@@ -45,6 +45,8 @@
     Route::get('talks/{id}/delete', 'TalksController@destroy');
     Route::get('conferences/{id}/delete', 'ConferencesController@destroy');
 
+    Route::get('talks/{talkId}/revisions/{revisionId}', 'TalkRevisionsController@show');
```

Comment:
Yah, the more I think about it--especially given how the talk versions controller ended up--i think I prefer query parameyer

Sent from Outlook




On Sat, Jun 6, 2015 at 7:38 AM -0700, "Adam Wathan" <notifications@github.com> wrote:












In app/Http/routes.php:
> @@ -45,6 +45,8 @@
>      Route::get('talks/{id}/delete', 'TalksController@destroy');
>      Route::get('conferences/{id}/delete', 'ConferencesController@destroy');
>  
> +    Route::get('talks/{talkId}/revisions/{revisionId}', 'TalkRevisionsController@show');



Should just use talk-revisions/{id} or similar, no need to nest, good read: http://weblog.jamisbuck.org/2007/2/5/nesting-resources



Or, like you suggested in Slack, talks/{id}?revision=2r89ce298, which I think I actually like the looks of better.



—
Reply to this email directly or view it on GitHub.